### ISSUE TITLE
PhantomJS 1.9.x incompatible with the Socket.io upgrade in sails@0.11.0

### ISSUE THREAD

**Issue by searls** [balderdashy/sails#2648]
Just a heads up to anyone curious, but while upgrading from sails@0.10 to sails@0.11, we found that the client-side `sails.io.js` upgrade resulted in all of our phantomjs-based integration tests hanging, with webdriver unable to connect to the page.

We resolved this by upgrading PhantomJS to 2.0

**Comment by tjwebb** [comment]
Is it that phantom 1.9 is just incompatible with socket.io 1.x?

**Comment by searls** [comment]
I haven't tested it separately but the presence of sails.io.js in the page does seem to break it. If that file is strictly socket.io, then yes.

**Comment by CWyrtzen** [comment]
Has all of the testing for this been completed to isolate the issue?

**Comment by sailsbot** [comment]
Thanks for posting, @searls.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
